lib/ansible/modules/cloud/amazon/cloudformation.py

```diff
@@ -500,6 +489,7 @@ def main():
 
 # import module snippets
 from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.ec2 import boto_exception
```

Comment:
Can you add this to the list on line 230 (and move line 491 to the top while you're there)